checker/src/main/java/org/checkerframework/checker/resourceleak/MustCallConsistencyAnalyzer.java

```diff
@@ -1241,6 +1242,28 @@ private void updateObligationsForAssignment(
     }
   }
 
+  /**
+   * Updates a set of obligations to account for a resource declaration in a try-with-resources
+   * tree. If the resource is a local variable (<em>not</em> a variable declaration), remove all
```

Comment:
I would change `the resource is a local variable (<em>not</em> a variable declaration)`  to `the resource is a use of a local variable`.